code/datums/elements/dextrous.dm

```diff
@@ -51,7 +51,9 @@
 /datum/element/dextrous/proc/on_hand_clicked(mob/living/hand_haver, atom/target, proximity, modifiers)
 	SIGNAL_HANDLER
 	if(!proximity)
-		return NONE
+		var/obj/item/can_be_obj = target
+		if(!istype(can_be_obj, /obj/item/storage) && !(can_be_obj.item_flags & IN_STORAGE))
```

Comment:
why are we checking if its a storage item?

Reply:
So that the proximity process does not consider the check as use on itself and skips the storage item so that it can be attacked (which is necessary to open it).

Reply:
why aren't you checking for the storage datum? objects like modsuits have storage while not being storage items

Reply:
Fikou is correct you will need to check if it has storage, not if it is a subtype of `item/storage`

Reply:
did it